doc/mzml-mzqc-example.md

```diff
@@ -18,7 +18,7 @@ Next, the metrics computed for the whole run can be be deposited as a child of t
 		<cvParam cvRef="QC" accession="QC:4000181" name="Charged spectra ratio +4 over +2" value="0.012987012987012988" />
 		<userParam name="mzml_id" type="xsd:string" value="20090810_SvNa_QC_BSA50fmol.RAW"/>
 ```
-Please note that for example brevity purposes the _mzML_ was truncated to 3 spectra, however the metrics were calculated on the whole to reflect realistic values.
+Please note that for example brevity purposes the _mzML_ was truncated to 3 spectra, however the metrics were calculated on the whole to reflect realistic values. Also, the NativeID format was truncated to the index of `spectrum=` bit.
```

Comment:
I don't think this latter part (truncating the nativeID) is actually valid. We should use complete examples instead.